setup.py

```diff
@@ -35,11 +35,18 @@
         ]
     },
     install_requires=[
+        'pillow',
         'pandas',
         'pymongo',
         'pyserial',
         'six',
-        'utm'
+        'scipy',
+        'utm',
+        'matplotlib',
+        'jupyter',
+        'ipyleaflet',
+        'sklearn',
+        'flask'
```

Comment:
Are there any other packages besides flask that we could drop if we remove the webserver, or are all of the rest of these required for this package?